src/vanilla_29_sdk/cgame/cg_syscalls.asm

```diff
@@ -0,0 +1,106 @@
+code
+
+equ	trap_Print							-1
+equ	trap_Error							-2
+equ	trap_Milliseconds					-3
+equ	trap_Cvar_Register					-4
+equ	trap_Cvar_Update					-5
+equ	trap_Cvar_Set						-6
+equ trap_Cvar_VariableStringBuffer		-7
+equ	trap_Argc							-8
+equ	trap_Argv							-9
+equ	trap_Args							-10
+equ	trap_FS_FOpenFile					-11
+equ	trap_FS_Read						-12
+equ	trap_FS_Write						-13 
+equ	trap_FS_FCloseFile					-14
+equ	trap_SendConsoleCommand				-15
+equ	trap_AddCommand						-16
+equ	trap_SendClientCommand				-17
+equ	trap_UpdateScreen					-18
+equ	trap_CM_LoadMap						-19
+equ	trap_CM_NumInlineModels				-20
+equ	trap_CM_InlineModel					-21
+equ	trap_CM_LoadModel					-22
+equ	trap_CM_TempBoxModel				-23
+equ	trap_CM_PointContents				-24
+equ	trap_CM_TransformedPointContents	-25
+equ	trap_CM_BoxTrace					-26
+equ	trap_CM_TransformedBoxTrace			-27
+equ	trap_CM_MarkFragments				-28
+equ	trap_S_StartSound					-29
+equ	trap_S_StartLocalSound				-30
+equ	trap_S_ClearLoopingSounds			-31
+equ	trap_S_AddLoopingSound				-32
+equ	trap_S_UpdateEntityPosition			-33
+equ	trap_S_Respatialize					-34
+equ	trap_S_RegisterSound				-35
+equ	trap_S_StartBackgroundTrack			-36
+equ	trap_R_LoadWorldMap					-37
+equ	trap_R_RegisterModel				-38
+equ	trap_R_RegisterSkin					-39
+equ	trap_R_RegisterShader				-40
+equ	trap_R_ClearScene					-41
+equ	trap_R_AddRefEntityToScene			-42
+equ	trap_R_AddPolyToScene				-43
+equ	trap_R_AddLightToScene				-44
+equ	trap_R_RenderScene					-45
+equ	trap_R_SetColor						-46
+equ	trap_R_DrawStretchPic				-47
+equ	trap_R_ModelBounds					-48
+equ	trap_R_LerpTag						-49
+equ	trap_GetGlconfig					-50
+equ	trap_GetGameState					-51
+equ	trap_GetCurrentSnapshotNumber		-52
+equ	trap_GetSnapshot					-53
+equ	trap_GetServerCommand				-54
+equ	trap_GetCurrentCmdNumber			-55
+equ	trap_GetUserCmd						-56
+equ	trap_SetUserCmdValue				-57
+equ	trap_R_RegisterShaderNoMip			-58
+equ	trap_MemoryRemaining				-59
+equ trap_R_RegisterFont					-60
+equ trap_Key_IsDown						-61
+equ trap_Key_GetCatcher					-62
+equ trap_Key_SetCatcher					-63
+equ trap_Key_GetKey						-64
+equ trap_PC_AddGlobalDefine				-65
+equ	trap_PC_LoadSource					-66

```

Comment:
should this be PC LOAD LETTER?

Reply:
Yes, I'm strongly considering changing it to `trap_PC_LoadLetter`.